lib/galaxy/webapps/galaxy/api/jobs.py

```diff
@@ -147,12 +142,23 @@ class JobIndexSortByEnum(str, Enum):
     description="Return jobs starting from this specified position. For example, if ``limit`` is set to 100 and ``offset`` to 200, jobs 200-299 will be returned.",
 )
 
+query_tags = [
+    IndexQueryTag("user", "The user email of the user that executed the Job.", "u"),
+    IndexQueryTag("tool_id", "The tool ID corresponding to the job.", "t"),
+    IndexQueryTag("runner", "The job runner name used to execte the job.", "r"),
+    IndexQueryTag("handler", "The job handler name used to execute the job.", "h"),
```

Comment:
This doesn't feel like something we should expose to non-admins, those are volatile and at the discretion of the admin. Can we exclude these from the schema ?